pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -71,32 +73,48 @@ def run(self):
-            advisories["image"] = self.create_advisory("RHEA", "image", "ga")
-        else:  # z-stream release
-            advisories["rpm"] = self.create_advisory("RHBA", "rpm", "standard")
-            advisories["image"] = self.create_advisory(
-                "RHBA", "image", "standard")
-        if self.release_version[0] > 3:
-            advisories["extras"] = self.create_advisory(
-                "RHBA", "image", "extras")
-            advisories["metadata"] = self.create_advisory(
-                "RHBA", "image", "metadata")
-        _LOGGER.info("Created advisories: %s", advisories)
-
-        _LOGGER.info("Saving the advisories to ocp-build-data...")
-        self.save_advisories(advisories)
-
-        _LOGGER.info("Sending an Errata live ID request email...")
-        self.send_live_id_request_mail(advisories)
-
-        _LOGGER.info("Creating a release JIRA...")
-        jira_issues = self.create_release_jira(advisories)
+        
+        if self.default_advisories:
+            advisories = self.get_advisories()
+        else:
+            _LOGGER.info("Creating advisories for release %s...", self.release_name)
+            advisories = {}
+
+            if self.release_version[2] == 0:  # GA release
+                advisories["rpm"] = self.create_advisory("RHEA", "rpm", "ga")
+                advisories["image"] = self.create_advisory("RHEA", "image", "ga")
+            else:  # z-stream release
+                advisories["rpm"] = self.create_advisory("RHBA", "rpm", "standard")
+                advisories["image"] = self.create_advisory(
+                    "RHBA", "image", "standard")
+            if self.release_version[0] > 3:
+                advisories["extras"] = self.create_advisory(
+                    "RHBA", "image", "extras")
+                advisories["metadata"] = self.create_advisory(
+                    "RHBA", "image", "metadata")
+            _LOGGER.info("Created advisories: %s", advisories)
+
+            _LOGGER.info("Saving the advisories to ocp-build-data...")
+            self.save_advisories(advisories)
+
+            _LOGGER.info("Sending an Errata live ID request email...")
+            self.send_live_id_request_mail(advisories)
+
+            _LOGGER.info("Creating a release JIRA...")
+            jira_issues = self.create_release_jira(advisories)
+
+            _LOGGER.info("Adding placeholder bugs to the advisories...")
+            for kind, advisory in advisories.items():
+                # don't create placeholder bugs for OCP 4 image advisory and OCP 3 rpm advisory
+                if (
+                    not advisory
+                    or self.release_version[0] >= 4
+                    and kind == "image"
+                    or self.release_version[0] < 4
+                    and kind == "rpm"
+                ):
+                    continue
+                self.create_and_attach_placeholder_bug(kind, advisory)
```

Comment:
In the future we can improve this logic by only adding placeholder bugs if not bugs are swept into an advisory to ensure advisories are ready for QE. But for now it seems to ok to only create placeholder bugs for new advisories.